sopel/plugins/rules.py

```diff
@@ -333,17 +337,17 @@ def from_callable(cls, settings, handler):
 
         :param settings: Sopel's settings
         :type settings: :class:`sopel.config.Config`
-        :param callable handler: Function-based rule handler
-        :return: An instance of this class created from the ``handler``
+        :param callable handler: a function-based rule handler
+        :return: an instance of this class created from the ``handler``
         :rtype: :class:`AbstractRule`
 
         Sopel's function-based rule handlers are simple callables, decorated
         with :mod:`sopel.module`'s decorators to add attributes, such as rate
         limit, threaded execution, output prefix, priority, and so on. In order
-        to load these functions as rule object, this class method can be used:
-        it takes the bot's ``settings`` and a clean ``handler``.
+        to load these functions as rule objects, this class method can be used;
+        it takes the bot's ``settings`` and a cleaned ``handler``.
 
-        A "clean handler" is a function, decorated appropriatly, and passed
+        A "cleaned handler" is a function, decorated appropriately, and passed
         through the filter of the
         :func:`loader's clean<sopel.loader.clean_callable>` function.
```

Comment:
I glossed over this one before, whoops.

```suggestion
        through the filter of the loader's
        :func:`cleaning function<sopel.loader.clean_callable>`.
```

Also I think we're pretty consistently using a space before `<` in these x-refs elsewhere, but I'm not too fussed about that.